### DISCUSSION TITLE
Using Sqlmodel with Flask

### DISCUSSION THREAD

**Discussion by dave42w** [fastapi/sqlmodel#1140] [Questions]
Is anyone using Sqlmodel with Flask? I started thinking I would use the flask-sqlalchemy extension but it doesn't let me use the Sqlmodel create engine.

Any examples of using Sqlmodel with Flask?

Thanks

**Comment by dave42w** [comment]
So it looks like I'm going to write a personal flask extension for sqlmodel. It seems the easiest way to use sqlmodel within a flask app. If there is interest I'll look to sharing it on pipy at some point.

**Reply by NathanByte001** [reply]
definitely!

**Reply by adalseno** [reply]
It would be interesting, even if I wonder if it's worthwhile. With the latest version, SQLModel `create_engine` is SQLAlchemy one. There is a difference in Session instead (SQLModel is different from SQLAlchemy, and Flask-SQLAlchemy is different too).

**Comment by NathanByte001** [comment]
Its not a common combination tho, but FIRST make sure you have the latest versions for both(simple stuffs like this go a long way)
